src/app/component/eco-news/create-news/drag-and-drop/drag-and-drop.component.html

```diff
@@ -0,0 +1,8 @@
+<div *ngIf="!files.length" class="dropzone" appDragAndDrop (files)="filesDropped($event)">
```

Comment:
Plaease, if you have more than 1 attribute - write them in separate lines

Reply:
fixed